packages/cli/src/index.ts

```diff
@@ -79,4 +80,16 @@ program
     return await devCommand(options, anonymousId);
   });
 
+program
+  .command('init')
+  .description(`Scaffold a new Novu application`)
+  .option(
+    '-s, --secret-key <secret-key>',
+    `Your Novu development environment secret key. Note that your novu app won't work outside of local mode without it.`
+  )
+  .option('-a, --api-url <url>', 'The Novu Cloud API URL', 'https://api.novu.co')
+  .action(async (options: IInitCommandOptions) => {
+    return await init(options, anonymousId);
+  });
+
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Small casing changes to maintain consistency, and replacing `Scaffold` with `create` as it's aligned with the previous `create-novu-app` naming. Should we also add a `.description` for the command to maintain consistency with the other commands having a description?‏

```suggestion
program
  .command('init')
  .description(`Create a new Novu application`)
  .option(
    '-s, --secret-key <secret-key>',
    `Your Novu development environment secret key. Note that your Novu app won't work outside of local mode without it.`
  )
  .option('-a, --api-url <url>', 'The Novu Cloud API URL', 'https://api.novu.co')
  .action(async (options: IInitCommandOptions) => {
    return await init(options, anonymousId);
  });

```